app_check/src/common/common.h

```diff
@@ -19,11 +19,18 @@ namespace firebase {
 namespace app_check {
 namespace internal {
 
+// Used by the function registry, so that other products can call App Check
+// functions without introducing a dependency.
 enum AppCheckFn {
   kAppCheckFnGetAppCheckToken = 0,
   kAppCheckFnCount,
 };
 
+// Helper function to get an existing AppCheck instance, since GetInstance will
+// create one if needed.
```

Comment:
Maybe rephrase to
// Helper function to get an existing AppCheck instance or nullptr
// if an instance does not already exist.

I think that should be sufficient to clarify what the method does differently from getInstance. But this is just a suggestion, feel free to ignore it if the existing comment is more clear to you.

Reply:
Yeah, makes sense. Fixed